<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QToolButton" name="toolButton_new">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QToolButton" name="toolButton_copy">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QToolButton" name="toolButton_paste">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <widget class="QToolButton" name="toolButton_open">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="2">
     <widget class="QToolButton" name="toolButton_cut">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" rowspan="2">
     <widget class="QToolButton" name="toolButton_save">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6" rowspan="2">
     <widget class="QToolButton" name="toolButton_approve">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="lang_ori">
      <property name="text">
       <string>Source</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="lang_tar">
      <property name="text">
       <string>Target</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QComboBox" name="lang_ori_select"/>
    </item>
    <item row="1" column="8">
     <widget class="QComboBox" name="lang_tar_select"/>
    </item>
    <item row="2" column="0" colspan="9">
     <widget class="QTableWidget" name="maintable">
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Source Language</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Target Language</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionNew_File"/>
    <addaction name="actionExport_Project"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionApprove"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="actionNope"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_File">
   <property name="text">
    <string>Import Project</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionNope">
   <property name="text">
    <string>Nope</string>
   </property>
  </action>
  <action name="actionApprove">
   <property name="text">
    <string>Approve Translation(s)</string>
   </property>
  </action>
  <action name="actionExport_Project">
   <property name="text">
    <string>Export Project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton_new</sender>
   <signal>clicked()</signal>
   <receiver>actionNew_File</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_copy</sender>
   <signal>clicked()</signal>
   <receiver>actionCopy</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_cut</sender>
   <signal>clicked()</signal>
   <receiver>actionCut</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_open</sender>
   <signal>clicked()</signal>
   <receiver>actionOpen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_paste</sender>
   <signal>clicked()</signal>
   <receiver>actionPaste</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_save</sender>
   <signal>clicked()</signal>
   <receiver>actionSave</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_approve</sender>
   <signal>clicked()</signal>
   <receiver>actionApprove</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
